src/PushoverMessage.php

```diff
@@ -230,6 +237,18 @@ public function highPriority()
         return $this->priority(self::HIGH_PRIORITY);
     }
 
+    /**
+     * Set the text type of the Pushover message to HTML.
+     *
+     * @return this
+     */
+    public function setHtml()
```

Comment:
Could you change this to a more descriptive method name? Something like `asHtml()` sounds good to me